eventmesh-starter/build.gradle

```diff
@@ -17,4 +17,7 @@
 
 dependencies {
     implementation project(":eventmesh-runtime")
-}
\ No newline at end of file
+    // Logging backend
+    runtimeOnly "org.apache.logging.log4j:log4j-core"
+    runtimeOnly "org.apache.logging.log4j:log4j-slf4j2-impl"
```

Comment:
Adding specific dependencies here is a bit inconsistent with the design here: https://eventmesh.apache.org/docs/instruction/runtime/#341-install-plugins

Since logging backend is shared within project, I hope that it can be added only once in `eventmesh-common` module~.

I searched for "log4j-core" within project, and it shows that some modules under `eventmesh-openconnect` is depending on it directly, which can be substituted with `eventmesh-common` denpendency.

Reply:
Most `log4j-core` dependencies should be in the `testRuntimeOnly` configuration, so they are invisible to external projects that use those `eventmesh-openconnect` artifacts.

Reply:
>it shows that some modules under `eventmesh-openconnect` is depending on it directly, which can be substituted with `eventmesh-common` denpendency.

What I mean is that `eventmesh-openconnect` should have depended on `eventmesh-common` to avoid having to manually add dependencies in `eventmesh-openconnect`. This was an oversight by previous developers. `eventmesh-openconnect` module should be less likely to be depended on by third-party applications.